### ISSUE TITLE
만약 이 html을 이어서 개발하신다면:)

### ISSUE THREAD

**Issue by rldudtj** [talita-aya/pinterest#2]
로그인 화면에서 이메일과 password를 입력하고 login을 했을 때,
실제 pinterest의 home 까지는 연동이 될 수 있었으면 좋겠습니다.

현재 pinterest의 실제 home에 연동이 되도 좋지만,
talita님의 아이디어와 실력이라면 그것을 기반으로 이 프로젝트만의 pinterest 기본 home 화면이 구현되어 로그인 후 연동되는 것이 훨씬 좋을 것 같습니다:)

제가 pinterest에 관심이 많기도 하고 이 프로젝트에 힘을 보태고 싶습니다.

When you log in after typing an email and password on the login screen,
I hope it can be linked to the actual home of the pinterest.

It may be linked to the actual home of the current pinterest.
Based on Talita's ideas and skills, it would be much better to implement the project's own pinnest home screen and link it after logging in. :)

I'm very interested in pinterest and I want to support this project.

**Comment by talita-aya** [comment]
Yes, I was thinking as you. I want to link everything in this login page to continue this project.
And feel free to support, I would love to have some help hahaha
